### ISSUE TITLE
Instructions on iterating an app are incomplete

### ISSUE THREAD

**Issue by wilhelm-si** [docker/docker.github.io#3824]
File: [get-started/part4.md](https://docs.docker.com/get-started/part4/), CC @johndmulhausen 

The sentence under

https://docs.docker.com/get-started/part4/#iterating-and-scaling-your-app

"Change the app behavior by editing code."

seems incomplete to me, since as far as I can tell, you also need to "docker build" with the correct tag and "docker push" to the registry, before a "docker stack deploy" will have the effect of actually updating your app behavior.

**Comment by joaofnfernandes** [comment]
Yup, you're right. The compose file mentions an image that was pushed to Docker Hub, so if you want to make changes to the code of your app you need to first build and push the new image, and only then run `docker stack deploy` again.

**Comment by mstanleyjones** [comment]
@gbarr01 you just need to rebuild the image after changing the code. :)
